challenge_scoring/io/streamlines.py

```diff
@@ -3,201 +3,116 @@
 
```

Comment:
Sorry I can't comment in the right file. In io.results, method load_results in not used anymore.